node-api/handlers/beacon/routes.go

```diff
@@ -151,11 +171,16 @@ func (h *Handler) RegisterRoutes(
 		{
 			Method:  http.MethodGet,
 			Path:    "/eth/v1/beacon/deposit_snapshot",
-			Handler: h.NotImplemented,
+			Handler: h.Deprecated,
+		},
+		{
+			Method:  http.MethodGet,
+			Path:    "/eth/v1/beacon/rewards/blocks/:block_id",
+			Handler: h.GetBlockRewards,
```

Comment:
is this right?

Reply:
yes the endpoint path is right, refer https://ethereum.github.io/beacon-APIs/#/Beacon/getBlockRewards